pkg/R/RDD.R

```diff
@@ -638,6 +638,66 @@ setMethod("minimum",
             reduce(x, min)
           })
 
+#' Get the mean value of an RDD's elements.
+#'
+#' @param x The RDD to get the mean value of elements from
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' rdd <- parallelize(sc, 1:10)
+#' meanStats(rdd) # 5.5
+#'}
+#' @rdname meanStats 
+#' @aliases meanStats,RDD
+setMethod("meanStats",
+          signature(x = "RDD"),
+          function(x) {
+            # build a new RDD which's elements are StatCounters c(mean, count)
+            newRdd <- lapply(x, function(x) { c(x, 1) })
+            reduce(newRdd, mergeStats)[1]
+          })
+
+#' Get the variance of the values of an RDD's elements.
+#'
+#' @param x The RDD to get the variance of the values of elements from
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' rdd <- parallelize(sc, 1:3)
+#' variance(rdd) # 0.6666667
+#'}
+#' @rdname variance 
+#' @aliases variance,RDD
+setMethod("variance",
```

Comment:
I don't know how the existing R functions are implemented. It seems to be very different between original base functions and these functions because of the Reduce used here.